### PR TITLE
Display disease name in dyadic disease name format

### PR DESCRIPTION
**Pull request by seeta-ramaraju**

### CODE REVIEW

File: templates/edit_entry.html.ep [hunk 1]
```diff
@@ -2,7 +2,7 @@
 %= include 'search'
 <div class="container"
   <div class="page-header">
-    <h1>Gene: <span class="text-primary"><i><%=$gfd->{gene_symbol}%></i>  <small>Disease: <span class="text-primary"><i><%=$gfd->{disease_name}%></i></span</small></h1>
+    <h1>Gene: <span class="text-primary"><i><%=$gfd->{gene_symbol}%></i>  <small>Disease: <span class="text-primary"><i><% if (index(lc $gfd->{disease_name}, "related") != -1 || index(lc $gfd->{disease_name}, "associated") != -1 || index(lc $gfd->{disease_name}, lc $gfd->{gene_symbol}) != -1) { %><%= $gfd->{disease_name} %><% } else { %><%= $gfd->{gene_symbol} %> - related <%= $gfd->{disease_name} %><% } %></i></span</small></h1>
```
Comment (dglemos) [inline templates/edit_entry.html.ep:5]: `&& index(lc $gfd->{disease_name}, lc $gfd->{gene_symbol}) != -1`

Example:
`12P12.5 INTRAGENIC DELETIONS ASSOCIATED WITH INTELLECTUAL DISABILITY`
would be displayed as `SOX5 - related 12P12.5 INTRAGENIC DELETIONS ASSOCIATED WITH INTELLECTUAL DISABILITY`

Comment (seeta-ramaraju) [reply templates/edit_entry.html.ep]: taken care

File: templates/show_attribs.html.ep [hunk 2]
```diff
@@ -2,7 +2,7 @@
 %= include 'search'
 <div class="container"
   <div class="page-header">
-    <h1>Gene: <span class="text-primary"><i><%=$gfd->{gene_symbol}%></i>  <small>Disease: <span class="text-primary"><i><%=$gfd->{disease_name}%></i></span></small></h1>
+    <h1>Gene2: <span class="text-primary"><i><%=$gfd->{gene_symbol}%></i>  <small>Disease: <span class="text-primary"><i><% if ((index(lc $gfd->{disease_name}, "related") != -1 || index(lc $gfd->{disease_name}, "associated") != -1) && index(lc $gfd->{disease_name}, lc $gfd->{gene_symbol}) != -1) { %><%= $gfd->{disease_name} %><% } else { %><%= $gfd->{gene_symbol} %> - related <%= $gfd->{disease_name} %><% } %></i></span></small></h1>
```
Comment (dglemos) [inline templates/show_attribs.html.ep:5]: Gene2 -> Gene

Comment (seeta-ramaraju) [reply templates/show_attribs.html.ep]: taken care in the latest commit

**Review by dglemos — COMMENTED** [review]
The update looks good.
The weirdest example I found is this one: https://www.ebi.ac.uk/gene2phenotype/gfd?dbID=42
After the update the disease is displayed as `COG7 - related COG7-CDG` which according to the new rules is correct but looks weird. 

The search only works with the original disease names - it could be improved in the future.

**Review by dglemos — APPROVED** [review]

**Review by dglemos — APPROVED** [review]

**Comment by dglemos** [comment]
Merged into release/110 and master

**Pull request merged**